Kordedekehine.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Salami Korede",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software,Open Source Enthusiast and Content Creator",
+  "links": [
+    {
+      "name": "check and follow me on Twitter",
+      "url": "https://twitter.com/akehindekorede",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "check and follow me on Github",
+      "url": "https://github.com/Kordedekehine",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "check and follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/salami-kehinde-korede-bb85281ab/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Check out my portfolio",
+      "url": "https://salamikorede.netlify.app/",
+      "icon": "FaWeb"
```

Comment:
@Kordedekehine This icon is not available in the library. Please double check that.
Here you can find available icons. Please refer: [Available icons](https://linkfree.eddiehub.io/icons)
Icon Suggestion: `FaLink`

Reply:
> @Kordedekehine This icon is not available in the library. Please double check that. Here you can find available icons. Please refer: [Available icons](https://linkfree.eddiehub.io/icons) Icon Suggestion: `FaLink`

```suggestion
      "icon": "FaLink" 
```